src/interfaces/tadidia.js

```diff
@@ -0,0 +1,42 @@
+const { Client, GatewayIntentBits } = require('discord.js');
+const { commands, commandPrefix } = require('../configs');
+require('dotenv').config();
+
+class Tadidia {
+    constructor() {
+        this.client = new Client({ intents: [GatewayIntentBits.MessageContent, GatewayIntentBits.Guilds, GatewayIntentBits.GuildMessages] });
+        this.init();
+    }
+
+    login = () => {
```

Comment:
classes should use `function` because this functions are actually methods

https://www.digitalocean.com/community/tutorials/understanding-arrow-functions-in-javascript
https://stackoverflow.com/questions/31095710/methods-in-es6-objects-using-arrow-functions